pilot/pkg/serviceregistry/kube/controller/ambient/authorization.go

```diff
@@ -432,7 +432,7 @@ const (
 	httpRuleFmt string = "ztunnel does not support HTTP rules (%s require HTTP parsing), in ambient mode you must use waypoint proxy to enforce HTTP rules. %s"
```

Comment:
```suggestion
	httpRuleFmt string = "ztunnel does not support HTTP attributes (%s field required HTTP parsing). In ambient mode you must use a waypoint proxy to enforce HTTP rules. %s"
```

Reply:
I don't like "HTTP rules" (unless I'm scribbling graffiti in 1996)  but I'm wary of making this text too long.
Otherwise, I might have ended with "to enforce policy on HTTP attributes"